File: clang/UKL/soal4.c
/*
Soal          : 4
Gambar        : B
Matrix kolom  : 5
*/

#include <stdio.h>

int main(){
    int A=6,
        B=3,
        kolom=5,
        u=A,
        s=B;
    printf("Deret Aritmatikanya adalah\n");
    for(int i=kolom;i>0;i--){ 
        for(int j=0;j<i;j++){
            printf("%d\t", u);
            u=u+B;
            s=s+u; 
        }
        printf("\n");
    }
    for(int i=kolom-1;i>0;i--){ 
        for(int j=kolom+1;j>0+i;j--){
            printf("%d\t", u);
            u=u+B;
            s=s+u; 
        }
        printf("\n");
    }
    s=s-u;
    printf("Jumlah Deret Aritmatikanya adalah %d\n", s);
}

File: python/XRPL_UKL1/soal4.py
#!/bin/python3
"""
Soal          : 4
Gambar        : B
Matrix kolom  : 5
"""


A=6
B=3
kolom=5
u=A
s=B
print("Deret Aritmatikanya adalah");
for i in range(kolom, 0, -1):
    for j in range(0, i):
        print(u, end="\t")
        u=u+B
        s=s+u
    print()

for i in range(kolom-1):
    for j in range(i+2):
        print(u, end="\t")
        u=u+B
        s=s+u
    print()

s=s-u
print(f"Jumlah Deret Aritmatikanya adalah {s}");
